doc/SONiC_202006_Release_Notes.md

```diff
@@ -0,0 +1,134 @@
+# SONiC 201911 Release Notes
+
+This document captures the new features added and enhancements done on existing features/sub-features for the SONiC 202006 release.
+
+
+
+# Table of Contents
+
+ * [Branch and Image Location](#branch-and-image-location)
+ * [Dependency Version](#dependency-version)
+ * [Security Updates](#security-updates)
+ * [Feature List](#feature-list)
+ * [SAI APIs](#sai-apis)
+ * [Contributors](#contributors)
+
+
+# Branch and Image Location  
+
+Branch : https://github.com/Azure/sonic-buildimage/tree/202006 <br>
+Image  : https://sonic-jenkins.westus2.cloudapp.azure.com/  (Example - Image for Broadcom based platforms is [here]( https://sonic-jenkins.westus2.cloudapp.azure.com/job/broadcom/job/buildimage-brcm-202006/lastSuccessfulBuild/artifact/target/))
+
+# Dependency Version
+
+|Feature                    | Version  |
+| ------------------------- | --------------- |
+| Linux kernel version      | linux_4.9.0-11-2 (4.9.189-3+deb9u2)   |
+| SAI   version             | SAI v1.6.3    |
+| FRR                       | 7.2    |
+| LLDPD                     | 0.9.6-1    |
+| TeamD                     | 1.28-1    |
+| SNMPD                     | 5.7.3+dfsg-1.5    |
+| Python                    | 3.6.0-1    |
+| syncd                     | 1.0.0    |
+| swss                      | 1.0.0    |
+| radvd                     | 2.17-2~bpo9+1    |
+| isc-dhcp                  | 4.3.5-2 ([PR2946](https://github.com/Azure/sonic-buildimage/pull/2946) )   |
+| sonic-telemetry           | 0.1    |
+| redis-server/ redis-tools | 5.0.3-3~bpo9+2    |
+
+
+# Security Updates
+
+1. Kernal upgraded from 4.9.110-3deb9u6 (SONiC Release 201904) to 4.9.168-1+deb9u5 in this SONiC release. 
+   Change log: https://tracker.debian.org/media/packages/l/linux/changelog-4.9.168-1deb9u5
+2. Docker upgraded from 18.09.2\~3-0\~debian-stretch to 18.09.8\~3-0\~debian-stretch. 
+   Change log: https://docs.docker.com/engine/release-notes/#18098 
+
+# Feature List
+
+#### Build Improvements 
+This document provides functional design and specifications of BFD protocol as defined in RFC 5880, 5881, 5882 and 5883.BFD protocol defines a method of rapid detection of the failure of a forwarding path by checking that the next hop router is alive. The protocol will be able to detect the forwarding path failure in milliseconds depending on the actual configuration. 
```

Comment:
This content (BFD) does not match the heading (Build Improvements). The topic here is the latter.

Reply:
Done.

Reply:
Ack